spec/api/v1/api_crops_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Api::V1::CropsController do
+
+  before(:each) do
+    @crop = FactoryGirl.create(:tomato)
+    @crop.save
```

Comment:
this .save is superfluous here... FG.create does that anyway.  But apart from that, let's use the newer RSpec syntax, "let", instead of "before :each".  Also, the tests following should use the expect(blah).to syntax, not blah.should.